gfsphysics/GFS_layer/GFS_land_perts.F90

```diff
@@ -0,0 +1,249 @@
+module GFS_land_perts
```

Comment:
Shouldn't this entire code be in the CCPP "stochastic_physics" group, either as an additional scheme or as part of the existing `GFS_stochastic` scheme?

Once we remove IPD, there will be no access to the soilveg data etc. anymore.

Reply:
Update: see my comment below for file `stochastic_physics/stochastic_physics_wrapper.F90`.